test/FunctionalTests/Tests/DataServices/UnitTests/ClientCSharpUnitTests/DataWebClientCSharp/LinqTests.cs

```diff
@@ -7899,7 +7899,7 @@ internal string ReturnString(string value)
             return value;
         }
 
-        // [TestMethod]
+        [TestMethod]

```

Comment:
@habbes I think this test could be simplified like this:
```
foreach (var option in new[] {"$custom", "$apply", "$skiptoken", "$delta"})
{
    var query = ((DataServiceQuery<League>)from l in context.CreateQuery<League>("Leagues") select l)
                                                .AddQueryOption(option, 1);

    try
    {
        query.GetEnumerator();
    }
    catch (Exception e)
    {
        if (e.Message != string.Format(System.Globalization.CultureInfo.InvariantCulture, "The query option {} is not supported.", option)
        {
            throw new Exception("Test Failed");
        }
    }
}
```

Reply:
I still tested $custom separately in order to make a distinction between unknown/custom and official query options.